todoapp/src/components/Todo.tsx

```diff
@@ -168,13 +184,83 @@ const useStyles = makeStyles({
         }
       }
     }
-  }
+  },
 })
 
+
+type Todo = {
+  id: string;
+  text: string;
+  completed: boolean
+}
+
 function Todos(): JSX.Element {
```

Comment:
at some point you're using arrow operator and here you're using function keyword. Make it consistent.

Reply:
It indicates, multiple devs worked on code.